src/pages/juniorPromiseRequest/JuniorPromiseRequestPage.tsx

```diff
@@ -78,28 +79,37 @@ const JuniorPromiseRequestPage = () => {
 
   const handlePostAppointment = () => {
     if (isAllSelected) {
-      postAppointment({
-        seniorId: seniorId,
-        topic: activeButton === '선택할래요' ? selectedButtons : [],
-        personalTopic: activeButton === '선택할래요' ? '' : inputVal,
-        timeList: [
-          {
-            date: selectedTime[0].clickedDay,
-            startTime: selectedTime[0].selectedTime.split('-')[0],
-            endTime: selectedTime[0].selectedTime.split('-')[1],
+      postAppointment(
+        {
+          seniorId: seniorId,
+          topic: activeButton === '선택할래요' ? selectedButtons : [],
+          personalTopic: activeButton === '선택할래요' ? '' : inputVal,
+          timeList: [
+            {
+              date: selectedTime[0].clickedDay,
+              startTime: selectedTime[0].selectedTime.split('-')[0],
+              endTime: selectedTime[0].selectedTime.split('-')[1],
+            },
+            {
+              date: selectedTime[1].clickedDay,
+              startTime: selectedTime[1].selectedTime.split('-')[0],
+              endTime: selectedTime[1].selectedTime.split('-')[1],
+            },
+            {
+              date: selectedTime[2].clickedDay,
+              startTime: selectedTime[2].selectedTime.split('-')[0],
+              endTime: selectedTime[2].selectedTime.split('-')[1],
+            },
+          ],
```

Comment:
```suggestion
          timeList: selectedTime.map(({ clickedDay, selectedTime }) => ({
            date: clickedDay,
            startTime: selectedTime.split('-')[0],
            endTime: selectedTime.split('-')[1],
          })),
```

p3) 이렇게 간결하게 중복코드 줄일 수 있을것같아용

Reply:
헙 너무 좋습니다!!! 이전 PR에 포함했습니다!!